roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
```

Comment:
The problem we are seeing in production is that root disks are getting filled up by data frameworks and/or mesos sandbox logs and causing node instability. Right now we allocate 40% of the attached disk to docker and the remainder of the disk is unavailable. So the first goal was actually to make the remainder of the external disk available for these things by default. As far as the logical volumes for docker, mesos, and mantldata, see #759 and #820. I know there is some flexibility wanted with docker to choose different file systems but maybe we could simplify with a single lv that is used for mesos and data.  thoughts?